packages/lib/src/utils.ts

```diff
@@ -311,22 +312,76 @@ export const underscores = (str: string): string => {
  * @param filename The filename to search for
  * @returns The file info
  */
-export const findFileInDirs = (dirs: string[], filename: string): FileInfo => {
-    const FileInfo: FileInfo = {
-        path: null,
-        filename,
-        exists: false,
+export const findFilesInDirs = async (dirs: string[], filename: string): Promise<FileInfo[]> => {
+    const filesInfo: FileInfo[] = []
+
+    const pattern = dirs.map((dir) => join(dir, filename))
+    const _files = await glob(pattern)
+
+    // Remove duplicates
+    const files = [...new Set(_files)]
+
+    for (const filePath of files) {
+        const fileInfo: FileInfo = {
+            path: null,
+            filename,
+            exists: false,
+        }
+        fileInfo.exists = existsSync(filePath)
+        if (fileInfo.exists) {
+            fileInfo.path = filePath
+            filesInfo.push(fileInfo)
+        }
+    }
+
+    if (filesInfo.length === 0) {
+        filesInfo.push({
+            path: null,
+            filename,
+            exists: false,
+        })
     }
-    for (const dir of dirs) {
-        const filePath = join(dir, filename)
-        FileInfo.exists = existsSync(filePath)
-        if (FileInfo.exists) {
-            FileInfo.path = filePath
-            return FileInfo
+
+    return filesInfo
+}
+
+/**
+ * Find a file in a list of directories (sync)
+ * @param dirs The directories to search in
+ * @param filename The filename to search for
+ * @returns The file info
+ */
+export const findFilesInDirsSync = (dirs: string[], filename: string): FileInfo[] => {
```

Comment:
It would be good to align on either using sync or async for file loading. We have several places with duplicate utilities for sync/async.

Reply:
I actually wanted to stay with async but there was one place where that would have led to a lot of conversion work

Reply:
oh yes, I think it was in the ejs templates, but it may be that async now also works in ejs, I can try it out

Reply:
Gotcha, might be worth testing async ejs in a future MR then.